### DISCUSSION TITLE
How to claim free frontend masters github promo?

### DISCUSSION THREAD

**Discussion by kuncisuksesberjualan002** [community/community#121398] [GitHub Education] [answered]
### Hi everyone,

Hello, i have problem, how to claim my free frontend masters course from github edu? anyone can help me?

**Comment by haululazkiyaa** [comment, marked as answer]
I have been try to claim frontend masters course before, you can try to register email in frontend masters with same email your github education, then register from this links: https://frontendmasters.com/welcome/github-student-developers/

**Comment by berthablea** [comment]
you can try to register email in frontend masters
